blueprints/-addon-import.js

```diff
@@ -4,18 +4,9 @@ const stringUtil = require('ember-cli-string-utils');
 const path = require('path');
 const inflector = require('inflection');
 
-const maybePolyfillTypeScriptBlueprints = require('./-maybe-polyfill-typescript-blueprints');
-
 module.exports = {
   description: 'Generates an import wrapper.',
 
-  shouldTransformTypeScript: true,
```

Comment:
`*-addon` blueprints (app re-exports) always need to generate JS files, so no need to try and transform TS -> JS for these.